docs/src/use/configure/parser.md

```diff
@@ -68,6 +68,33 @@ export default [
 ];
 ```
 
+Parsers also refer external config files. Here's an example of using `babel.config.json` for Babel ESlint parser:
+
+```json
+// babel.config.json
+{
+     presets: ["@babel/preset-env"]
+}
+```
+
+```js
+// eslint.config.js
+import babelParser from "@babel/eslint-parser";
+
+export default [
+    {
+        languageOptions: {
+            parser: babelParser,
+            parserOptions: {
+                babelOptions: {
+                  configFile: "./babel.config.json"
+                }
+            }
+        }
+    }
+];
+```
```

Comment:
I think these details are parser-specific and should be looked for in the documentation of the parser as suggested in the paragraph above:

https://github.com/eslint/eslint/blob/7bcda760369c44d0f1131fccaaf1ccfed5af85f1/docs/src/use/configure/parser.md?plain=1#L48

Reply:
Totally aligned. As eslint chose babel as example to showcase using a parser, devs used to simply copy paste the config from eslint itself. Keeping this in mind, I added few more details keeping it very minimal by not delving deep into babel (devs can easily get them from official docs).

Reply:
Wouldn't it be better to add a link to the [docs of the Babel ESLint parser](https://www.npmjs.com/package/@babel/eslint-parser) and update the code examples there? They have an example that uses `configFile` with a path ([here](https://github.com/babel/babel/blob/419644f27c5c59deb19e71aaabd417a3bc5483ca/eslint/babel-eslint-parser/README.md?plain=1#L80-L92)), but it's for eslintrc.